Integrations/src/main/java/io/deephaven/integrations/learn/Future.java

```diff
@@ -1,20 +1,19 @@
 package io.deephaven.integrations.learn;
 
 import io.deephaven.db.v2.sources.ColumnSource;
-import io.deephaven.integrations.python.PythonFunctionCaller;
-import org.jpy.PyObject;
+import java.util.function.Function;
 
 /**
  * Future performs a deferred computation on a portion of a table.
  */
 class Future {
 
-    private final PyObject func;
+    private final Function<Object[], Object> func;
+    private final ColumnSource<?>[][] colSet;
```

Comment:
Rename to colSets.  I think that is more clear than colSet, since you have multiple sets.